flair/nn/model.py

```diff
@@ -572,6 +589,7 @@ def __init__(
         decoder: Optional[torch.nn.Module] = None,
         inverse_model: bool = False,
         train_on_gold_pairs_only: bool = False,
+        should_embed_sentence: bool = True,
```

Comment:
Why is this parameter added? Are there models that inherit from DefaultClassifier that do not embed?